packages/account-v2/src/utils/accountLimitsUtils.ts

```diff
@@ -0,0 +1,140 @@
+import { GetLimits } from '@deriv/api-types';
+import { FormatUtils } from '@deriv-com/utils';
+import { TCurrency } from '../types';
+
+export const CATEGORY = {
+    footer: 'footer',
+    header: 'header',
+    submarket: 'submarket',
+} as const;
+
+export type TAccountLimitValues = {
```

Comment:
**NIT**: If this is a sharable type can we move it to types.ts?

Reply:
@likhith-deriv just checking if it is only used in accountLimits components , do we need to add it in types or can be scoped it here?